### PR TITLE
files from Tinder study

### PR DESCRIPTION
**Pull request by bwliv**

### CODE REVIEW

**Review by bojum — CHANGES_REQUESTED** [review]
Please remove tinder.html. See [1.3.5](https://jtr13.github.io/cc19/index.html#submission-steps)

**Comment by bwliv** [comment]
thanks for taking care of this! doing it myself would have been complicated so i appreciate it

**Comment by bwliv** [comment]
Did this work? Want to double check. Thanks!

**Comment by bojum** [comment]
No problem! And thanks for checking. We are refactoring the way files are organized right now so don't worry about 'all checks have failed'.

File: tinder_study.Rmd [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+    panel.grid.major = element_line(color='white'),
+    panel.grid.minor = element_line(color='white'),
+    panel.border = element_rect(linetype = 'twodash',color=tinder_pink,fill=NA),
+    strip.text = element_text(color='white',face='bold'),
+    strip.background = element_rect(fill=tinder_pink),
+    plot.title = element_text(hjust = 0.5)
+  )
+}
+
+bentinder = devtools::install_github()
```
Comment (skyetim) [inline tinder_study.Rmd]: What’s the repo?

**Review by skyetim — CHANGES_REQUESTED** [review]

**Comment by bwliv** [comment]
That can be removed. Fragment of unused code.

**Comment by skyetim** [comment]
Please remove it as it causes the failure of building. I would suggest going through all the codes again to remove all redundant codes.

**Comment by bwliv** [comment]
I deleted my local repo - can you just remove that line? It's the only necessary change I can see. Thanks!

**Comment by skyetim** [comment]
Okay

**Comment by bojum** [comment]
Remove the line causes a new error. `bentinder` object is not initialized before it is called in line 33. 
 Missing anything, @bwliv?

**Comment by bwliv** [comment]
Yes, since bentinder is in .csv in resources I don't know where to read the data frame from - that I will need you to add. bentinder.csv should be in the resources. I can't do it because I don't know how bookdown loads resources. wasn't sure if it was the same as the normal command to load from the working directory. if so it just needs a bentinder = load.csv('bentinder.csv')

**Comment by bojum** [comment]
🎉 merged!! Congrats and thank you for your contribution!!

**Comment by skyetim** [comment]
all the photos are down. please change all the location like bentinder.csv

**Pull request merged**